lamvery/actions/api.py

```diff
@@ -0,0 +1,262 @@
+                        'Access-Control-Allow-Headers': {'type': 'string'},
+                    }
+                }
+            },
+            'x-amazon-apigateway-integration': {
+                'responses': {
+                    'default': {
+                        'statusCode': '200',
+                        'responseParameters': {
+                            'method.response.header.Access-Control-Allow-Methods': cors.get('methods'),
+                            'method.response.header.Access-Control-Allow-Headers': cors.get('headers'),
+                            'method.response.header.Access-Control-Allow-Origin': cors.get('origin'),
+                        }
+                    }
+                },
+                'requestTemplates': {
+                    'application/json': '{"statusCode": 200}'
+                },
+                'type': 'mock'
+            }
+        }
+
+    def _apply_api(self, client, api_id, api_conf):
+        api = client.get_rest_api(api_id)
+        ret = None
+
+        if api is None:
+            if self._remove:
+                self._logger.info('"{}" is not exists.'.format(api_id))
+            else:
+                self._logger.info('[API] Add new rest api.')
+                ret = client.import_rest_api(api_conf)
+        else:
+            if self._remove:
+                self._logger.warn('[API] Delete rest api "{}".'.format(api_id))
+                client.delete_rest_api(api_id)
+            else:
+                self._logger.info('[API] Update rest api "{}".'.format(api_id))
+                ret = client.put_rest_api(api_id, api_conf)
+
+        return ret
+
+    def _print_apply_result(self, ret):
+        if ret is not None:
+
+            self._logger.info('[API] ID: {}'.format(ret['id']))
+            self._logger.info('[API] Name: {}'.format(ret['name']))
+            self._logger.info('[API] Description: {}'.format(ret.get('description')))
+
+            for w in ret.get('warnings', []):
+                self._logger.warn('[API] Warning: "{}".'.format(w))
+
+    def _deploy(self, client, api_id, stage):
+        return client.create_deployment(api_id, stage)
+
+    def _print_deploy_result(self, ret):
+        if ret is not None:
+            self._logger.info('[API] Deployment ID: {}'.format(ret['id']))
+            self._logger.info('[API] Deployment description: {}'.format(ret.get('description')))
+            self._logger.info('[API] Deployment API Summary: {}'.format(ret.get('apiSummary')))
+
+    def _print_conf_diff(self, local, remote):
+        str_diff = diff(remote, json.loads(json.dumps(local))).__str__()
+        str_diff = re.compile(r'^( +)([\-\+])', re.MULTILINE).sub(r'\2\1', str_diff)
+        self._logger.warn('[API] Configuration:\n{}'.format(
+            highlight(
+                str_diff,
+                get_lexer_by_name('diff'),
+                TerminalFormatter())
+            ))
```

Comment:
__[flake8]__
* [E123] closing bracket does not match indentation of opening bracket's line